abt-me-section

nectakin opened the pull request.

nectakin committed:
abt-me-section

nectakin committed:
small upd

треба викачати і додати у спрайт stroke (розділяюча лінія між about i role), інакше воно не відображатиметься у проєкті. аналогічно з зображеннями

nectakin committed:
upd-fixed mistakes

виправлено помилки.необхідно викачати і додати stroke і фото. також питання по icon-arrows-right. не до кінця працює fill через що вона трохи негарно виглядає. а також чомусь не виходить змінити розмір.

instreamer1 reviewed (commented):
В html  DOCTYPE and  body не нуженs. Семантика : 
 <section ">
  <div class="container"> content</div >
</section ">
Если боковых падингов нет container можно пропустить.
Путь к svg лучше так "./img/svg_sprite-opt.svg#icon-service_art" иначе может не отобразиться на странице.
Вектор добавил в main id="icon-vector"

nectakin committed:
fix mistakes

коли прибираю тег body-зникає повністю div "content" (закоментовано як app button). теги усі закриті, але проблема не фікситься. якщо можете переглянути у чому проблема, буду дуже вдячна!

також icon-vector має підтягнутись коли секція вже змаржиться повністю. аналогічно з фото якщо воно буде додане в img

instreamer1 reviewed (approved):
есть что доработать
